oojs.js

```diff
@@ -4,9 +4,31 @@
   // A favorites property that is initialized at 0. This cannot be set by user input.
   // An addSong method that adds a song to the songs property.
 
+class Playlist {
+  constructor(title) {
+    this.title = title
+    this.songs = []
+    this.favorites = 0
+  }
+  addSong(song) {
+    this.songs.push(song)
+  }
+}
+
 // Create an instance of the Playlist class.
 
+let popCrush = new Playlist('Pop Crush')
```

Comment:
:+1: although needs to be updated to take a `songs` argument. Or you could use default arguments to make `songs` be an empty array if no user input is passed in.